src/Components/QuestionList/QuestionListS.scss

```diff
@@ -3,8 +3,8 @@
 .QuestionList {
     display: flex;
     flex-direction: column;
-    width: 81vw;
-    height: 80vh;
+    width: 81%;
+    height: 80%;
```

Comment:
I think you are really going to back yourself into a wall by setting all these manual widths and heights. This can be dangerous, especially with percentages or vh and vw